Arduino/Actuator/Aditi Nikam/dcmotor.ino

```diff
@@ -0,0 +1,41 @@
+int motorPin = 2;
+int input1 = 8;
+int input2 = 9;
+void setup() {
+  Serial.begin(9600);
+  pinMode(motorPin,OUTPUT);
+  pinMode(input1,OUTPUT);
```

Comment:
Where are these input pins connected?

Reply:
2, 8 and 9 digital pin

Reply:
I meant what is connected to pin numbers 8 and 9... sorry the question wasn't clear 😛

Reply:
Ooh sorry! 8 and 9 are connected to input pins which decides the direction of the motor and 2 is connected for the speed.

Reply:
Ohkkk cool... Noice